src/CMakeLists.txt

```diff
@@ -96,7 +96,11 @@ if(BUILD_UNIT_TEST)
 endif(BUILD_UNIT_TEST)
 
 # link library dependencies
-target_link_libraries(${SERVER_OBJ_LIB_NAME} PUBLIC jwt-cpp jsonpath jsoncons ${CMAKE_THREAD_LIBS_INIT})
+# Setting max stack size explicetely to current default value of glibc on Ubuntu, to force MUSL
+# builds using the same max. Otherwise you might have hard to debug differences between MUSL and
+# builds.
+# See also https://wiki.musl-libc.org/functional-differences-from-glibc.html#Thread-stack-size
+target_link_libraries(${SERVER_OBJ_LIB_NAME}  PUBLIC jwt-cpp jsonpath jsoncons ${CMAKE_THREAD_LIBS_INIT} -Wl,-z,stack-size=8388608)
```

Comment:
I would like to use [`target_link_options`](https://cmake.org/cmake/help/v3.13/command/target_link_options.html). But it is available since cmake 3.13 and in our cmake files, the required minimal version is 3.12... But it works, therefore, I will approve it.